src/main/java/org/carlspring/strongbox/janusgraph/app/Application.java

```diff
@@ -16,6 +17,7 @@
           JanusGraphConfig.class,
           GremlinServerConfig.class,
           RepositoriesConfig.class })
+@ComponentScan(basePackages = {"org.carlspring.strongbox.janusgraph"})
```

Comment:
@cbrianhill can you please create configuration class for controllers instead of this `ComponentScan`?

Reply:
fixed